students/dychowicz_monika/lesson_05/lesson_05.py

```diff
@@ -0,0 +1,99 @@
+def greater_than_neighbours():
+    int_list = input("Enter a list of integer numbers separate by space: ")
+    new_list = int_list.split()
+    quanity = 0
+    if len(new_list) >= 3:
+        for i in range(1, len(new_list) - 1):
+            if new_list[i] > new_list[i + 1]:
+                quanity += 1
+    print("Numbers of elements greater than neighbours: ", quanity)
+
+
+def swap_min_and_max():
+    a = [int(s) for s in input("Give a list of unique integer  numbers seperate by space: ").split()]
+    index_of_min = 0
+    index_of_max = 0
+    for i in range(1, len(a)):
```

Comment:
iterating over all to just find max values is not optimal solution.
it's better to use `max` and `min` built-in functions.
could use it with conjunction with `.indexof`